src/LiveDevelopment/MultiBrowserImpl/documents/LiveHTMLDocument.js

```diff
@@ -231,8 +231,9 @@ define(function (require, exports, module) {
         // edit this file or set a conditional breakpoint at the top of this function:
         //     "this._debug = true, false"
         if (this._debug) {
```

Comment:
Similar to above, let's do `this._debug = false;` somewhere above and have this get skipped in release builds, but we can enable later if we need it.